### PR TITLE
[ticket/16172] Add group rank label to group view

### PR DESCRIPTION
**Pull request by senky**
PHPBB3-16172

Before:
Snímka obrazovky 2019-10-04 o 14 19 30

After:
Snímka obrazovky 2019-10-04 o 14 19 39

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [ ] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16172

### CODE REVIEW

**Comment by mrgoldy** [comment]
Will this not look weird with a rank image?

**Comment by dark-1** [comment]
I think this should be like :
```php
<!-- IF RANK_IMG or GROUP_RANK -->{{ lang('GROUP_RANK') ~ lang('COLON') }}<!-- ENDIF -->
<!-- IF RANK_IMG -->{RANK_IMG}<!-- ENDIF -->
<!-- IF GROUP_RANK -->{GROUP_RANK}<!-- ENDIF -->
```

**Comment by senky** [comment]
I don't think so. But you are right that it is weird with rank image. I decided to display the label only when the image is not present (because it is the only situation when it is unclear what the text is about).

File: phpBB/styles/prosilver/template/memberlist_body.html [hunk 1]
```diff
@@ -29,7 +29,12 @@ <h2 class="group-title"<!-- IF GROUP_COLOR --> style="color:#{GROUP_COLOR};"<!--
 			<!-- IF AVATAR_IMG -->{AVATAR_IMG}<!-- ENDIF -->
 			{% EVENT memberlist_body_group_rank_before %}
 			<!-- IF RANK_IMG -->{RANK_IMG}<!-- ENDIF -->
-			<!-- IF GROUP_RANK -->{{ lang('GROUP_RANK') ~ lang('COLON') }} {GROUP_RANK}<!-- ENDIF -->
+			<!-- IF GROUP_RANK -->
+				<!-- IF not RANK_IMG -->
+					{{ lang('GROUP_RANK') ~ lang('COLON') }}
+				<!-- ENDIF -->
+				{GROUP_RANK}
+			<!-- ENDIF -->
```
Comment (3D-I) [inline phpBB/styles/prosilver/template/memberlist_body.html]: Twig should be used on every bit of code added or amended.

**Pull request merged**